Dockerfile

```diff
@@ -7,11 +7,11 @@ COPY . .
 RUN CGO_ENABLED=0 go build -ldflags="-w -s -X 'main.version=${VERSION}'" ./cmd/image-registry-metrics-exporter
 
 FROM docker.io/alpine:3.18.0
+ARG CA_CERTIFICATES_VERSION=20230506-r0
 
 COPY --from=builder /build/image-registry-metrics-exporter /image-registry-metrics-exporter
 
-RUN apk update && apk add --no-cache ca-certificates=20230506-r0 \
-    && update-ca-certificates
+RUN apk update && apk add --no-cache ca-certificates=${CA_CERTIFICATES_VERSION}
```

Comment:
```suggestion
RUN apk upgrade --no-cache && apk add --no-cache ca-certificates=${CA_CERTIFICATES_VERSION}
```

Reply:
I want to keep the image as immutable and small as possible. So i don't have an interest in updating the packages. That's why i prefer to do an `apk update`

Reply:
But why running `apk update` then `apk --no-cache` ?